concrete/blocks/express_form/form.php

```diff
@@ -9,11 +9,12 @@
     array('form-options', t('Options')),
 ));?>
 </div>
-
+<?php if (isset($bOriginalID)):?>
```

Comment:
an express block form added since scrapbook without modification its view is displayed via the view of the internal block scrapbook, from where a useful variable $ bOriginalID will be available in the view.This variable will allow us to identify if the data is submitted from the block itself or from a view of a scrapbook